db/tag.c

```diff
@@ -6750,7 +6750,14 @@ void freedb_int(dbtable *db, dbtable *replace)
     }
 
     if (replace) {
+        pthread_rwlock_t lk_copy;
+        // Make a copy of original sc_live_lk
+        memcpy(&lk_copy, &db->sc_live_lk, sizeof(pthread_rwlock_t));
```

Comment:
since it's a struct maybe we could assign the value (ie lk_copy = db->sc_live_lk)?